### ISSUE TITLE
Create a new job to test the daily OL docker image

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guides-common#265]
Create a new job to test the daily OL docker image
- pick the OL docker image daily build
- use any OL daily build (if possible, use the same build level)
- test the guides which use OL docker image, e.g. getting start, and all kube guides

**Comment by evelinec** [comment]
- Andy will update the docker file to include build number information, which will take him few days
- pending on that for now

**Comment by evelinec** [comment]
Some review feedback on this automation job: 

Jenkins: 

- [ ] Add a description of this job, stating what this job is all about, the steps to run this job, and described the results. Include any specifics that are needed for this job. 

- [ ] The parameter nameS, ie, JDKLEVEL, JDK8. I think they are meant to specify the jdk levels that test are to be run with. Actually I think with just the JDKLEVEL parameter name, you can specify more jdk levels to run the test, ie, "openjdk11, oraclejdk8". Try it out. Otherwise, make the parameter names consistent? 

- [ ] I really think the build command script need to be version control under the `olguides-automation` repo.

**Comment by gkwan-ibm** [comment]
- [ ] need a doc
- [ ] presentation to the team

**Comment by evelinec** [comment]
- doc is finishing up this week

**Comment by gkwan-ibm** [comment]
- draft of the doc is completed

**Comment by evelinec** [comment]
Recently found that this job cannot be rerun with the same parameters, if the docker image that a job was run with is not within the 4 hours period. ie, the failed job #291 couldn't be rerun. This is not good for problem investigation and determination.

**Comment by andrewdes** [comment]
> This is not good for problem investigation and determination.

@evelinec yes I agree. I can look into ways to make the jobs that actually run the tests reproducible.

**Comment by gkwan-ibm** [comment]
- doc is completed

**Comment by andrewdes** [comment]
Opened the above in a separate issue: https://github.com/OpenLiberty/guides-common/issues/307. Closing this issue since the testing daily OL docker image job is complete.
